### ISSUE TITLE
Improve headers after #37

### ISSUE THREAD

**Issue by olemis** [skycoin/docs#45]
<img width="649" alt="screen shot 2018-02-21 at 9 45 02 am" src="https://user-images.githubusercontent.com/538253/36486267-1fbac12c-16ec-11e8-9b84-ab1f8613aa15.png">

Improvements in topic headings :

- [x] Ensure there is no duplication (e.g. `P2P network` included twice)
- [x] Remove pipe chars (i.e. `|` in subheading links)
- [x] Move all items in `content` folder into a new subfolder `en` folder for English docs
- [x] Change `Commit` in subhead links to be `History`
- [x] Change `Report` in subhead links to be `Report issue`
- [x] Ensure that `Edit` and `History` links (right now broken) will be fixed after adding `en` folder
- [x] Ensure that the URL in new issue body (click on `Report issue`) is correct once `en` folder will be added
- [x] Add `Discuss` subheader link pointing at https://t.me/Skycoin.
